src/mono-runtimes/mono-runtimes.targets

```diff
@@ -128,6 +129,16 @@
         Include="@(_MonoUtility->'$(_MSBuildDir)\%(Filename).pdb')"
     />
   </ItemGroup>
+  <ItemGroup>
+    <_MonoBuildUtility Include="mono-api-html.exe" />
+    <_MonoBuildUtility Include="mono-api-info.exe" />
+    <_MonoBuildUtility Include="pdb2mdb.exe" />
```

Comment:
With the "we need files in the mono bundle" idea in mind, `pdb2mdb.exe` should be added to the `@(_MonoUtility)` item group, ~line 108, alongside `mkbundle.exe` and `mono-symbolicate.exe`.